### PR TITLE
SOLR-162680: toString is not required when crafting a string message

### PR DESCRIPTION
**Pull request by epugh**
https://issues.apache.org/jira/browse/SOLR-162680

# Description

We sprinkle in a lot of our messages in the tests a lot of `.toString()`, however we don't need them.  They make the methods longer to read.

# Solution

Strip out toString() that aren't needed.

# Tests

Please describe the tests you've developed or run to confirm this patch implements the feature or solves the problem.

# Checklist

Please review the following and check all that apply:

- [x ] I have reviewed the guidelines for [How to Contribute](https://wiki.apache.org/solr/HowToContribute) and my code conforms to the standards described there to the best of my ability.
- [ x] I have created a Jira issue and added the issue ID to my pull request title.
- [ x] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [ x] I have developed this patch against the `main` branch.
- [ ] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

File: solr/core/src/test/org/apache/solr/search/ReturnFieldsTest.java [hunk 1]
```diff
@@ -461,7 +461,7 @@ public void testWhiteboxSolrDocumentConversion() {
             new SolrReturnFields(req("fl", allFieldNames + ",bogus1,bogus2,bogus3")))) {
 
       docOut = convertLuceneDocToSolrDoc(docIn, schema, rf);
-      final String debug = rf.toString() + " => " + docOut.toString();
+      final String debug = rf + " => " + docOut.toString();
```
Comment (risdenk) [inline solr/core/src/test/org/apache/solr/search/ReturnFieldsTest.java:464]: `docOut.toString()` still needed?

File: solr/core/src/test/org/apache/solr/search/ReturnFieldsTest.java [hunk 2]
```diff
@@ -435,7 +435,7 @@ public void testWhiteboxSolrDocumentConversion() {
             new SolrReturnFields(req("fl", "id,xxx:[geo f=store],uniq,foo_2_s1,subword")),
             new SolrReturnFields(req("fl", "id,xxx:subword,uniq,yyy:foo_2_s1,[geo f=store]")))) {
       docOut = convertLuceneDocToSolrDoc(docIn, schema, rf);
-      final String debug = rf.toString() + " => " + docOut.toString();
+      final String debug = rf + " => " + docOut.toString();
```
Comment (risdenk) [inline solr/core/src/test/org/apache/solr/search/ReturnFieldsTest.java:438]: `docOut.toString()` still needed?

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java [hunk 3]
```diff
@@ -538,7 +538,7 @@ public void testJsonRangeFacetWithSubFacet() throws Exception {
 
       } catch (AssertionError | RuntimeException ae) {
         throw new AssertionError(
-            solrQuery.toString() + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
+            solrQuery + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
```
Comment (risdenk) [inline solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:541]: `rsp.toString()` still needed?

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java [hunk 4]
```diff
@@ -615,7 +615,7 @@ public void testJsonRangeFacetAsSubFacet() throws Exception {
         }
       } catch (AssertionError | RuntimeException ae) {
         throw new AssertionError(
-            solrQuery.toString() + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
+            solrQuery + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
```
Comment (risdenk) [inline solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:618]: `rsp.toString()` still needed?

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java [hunk 5]
```diff
@@ -365,7 +365,7 @@ public void testJsonFacetRangeOfAsymmetricRates() throws Exception {
...
```
Comment (risdenk) [inline solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:368]: `rsp.toString()` still needed?

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java [hunk 6]
```diff
@@ -306,7 +306,7 @@ public void testFacetRangeOfAsymmetricRates() throws Exception {
...
```
Comment (risdenk) [inline solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:309]: `rsp.toString()` still needed?

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java [hunk 7]
```diff
@@ -239,7 +239,7 @@ public void testSimpleRangeFacetsOfSymmetricRates() throws Exception {
           }
         } catch (AssertionError | RuntimeException ae) {
           throw new AssertionError(
-              solrQuery.toString() + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
+              solrQuery + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
```
Comment (risdenk) [inline solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:242]: `rsp.toString()` still needed?

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java [hunk 8]
```diff
@@ -181,7 +181,7 @@ public void testSimpleRangeFacetsOfSymmetricRates() throws Exception {
...
```
Comment (risdenk) [inline solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:184]: `rsp.toString()` still needed?

File: solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java [hunk 9]
```diff
@@ -173,7 +173,7 @@ public void testInclude_Lower() throws Exception {
 
           } catch (AssertionError | RuntimeException ae) {
             throw new AssertionError(
-                solrQuery.toString() + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
+                solrQuery + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
```
Comment (risdenk) [inline solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java:176]: `rsp.toString()` still needed?

**Review by risdenk — APPROVED** [review]

File: solr/core/src/test/org/apache/solr/cloud/TestTolerantUpdateProcessorRandomCloud.java [hunk 10]
```diff
@@ -262,9 +262,7 @@ public void testRandomUpdates() throws Exception {
         }
       }
       assertEquals(
-          "expected error count sanity check: " + req.toString(),
-          expectedErrorsCount,
-          expectedErrors.size());
+          "expected error count sanity check: " + req, expectedErrorsCount, expectedErrors.size());
```
Comment (cpoerschke) [inline solr/core/src/test/org/apache/solr/cloud/TestTolerantUpdateProcessorRandomCloud.java:265]:
```suggestion
          "expected error count check: " + req, expectedErrorsCount, expectedErrors.size());
```

File: solr/core/src/test/org/apache/solr/cloud/api/collections/PurgeGraphTest.java [hunk 11]
```diff
@@ -46,9 +46,7 @@ public void setUpRepo() throws Exception {
     repository = new LocalFileSystemRepository();
     baseLocationUri =
         repository.createDirectoryURI(
-            createTempDir("backup_files_" + UUID.randomUUID().toString())
-                .toAbsolutePath()
-                .toString());
+            createTempDir("backup_files_" + UUID.randomUUID()).toAbsolutePath().toString());
```
Comment (cpoerschke) [inline solr/core/src/test/org/apache/solr/cloud/api/collections/PurgeGraphTest.java:49]: Did you mean to remove the `toString` here?
```suggestion
            createTempDir("backup_files_" + UUID.randomUUID()).toAbsolutePath());
```

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java (hunk 8 above)
Comment (cpoerschke) [reply solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:184]:
```suggestion
              solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java (hunk 7 above)
Comment (cpoerschke) [reply solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:242]:
```suggestion
              solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java (hunk 6 above)
Comment (cpoerschke) [reply solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:309]:
```suggestion
            solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java (hunk 5 above)
Comment (cpoerschke) [reply solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:368]:
```suggestion
            solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java (hunk 3 above)
Comment (cpoerschke) [reply solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:541]:
```suggestion
            solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

File: solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java (hunk 4 above)
Comment (cpoerschke) [reply solr/core/src/test/org/apache/solr/search/CurrencyRangeFacetCloudTest.java:618]:
```suggestion
            solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

File: solr/core/src/test/org/apache/solr/search/ReturnFieldsTest.java (hunk 2 above)
Comment (cpoerschke) [reply solr/core/src/test/org/apache/solr/search/ReturnFieldsTest.java:438]:
```suggestion
      final String debug = rf + " => " + docOut;
```

File: solr/core/src/test/org/apache/solr/search/ReturnFieldsTest.java (hunk 1 above)
Comment (cpoerschke) [reply solr/core/src/test/org/apache/solr/search/ReturnFieldsTest.java:464]:
```suggestion
      final String debug = rf + " => " + docOut;
```

File: solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java (hunk 9 above)
Comment (cpoerschke) [reply solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java:176]:
```suggestion
                solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

File: solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java [hunk 12]
```diff
@@ -220,7 +220,7 @@ public void testInclude_Lower_Gap2() throws Exception {
...
```
Comment (cpoerschke) [inline solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java:223]:
```suggestion
                solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

File: solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java [hunk 13]
```diff
@@ -267,7 +267,7 @@ public void testInclude_Lower_Gap2_hardend() throws Exception {
...
```
Comment (cpoerschke) [inline solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java:270]:
```suggestion
                solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

File: solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java [hunk 14]
```diff
@@ -327,7 +327,7 @@ public void testInclude_Upper() throws Exception {
 
         } catch (AssertionError | RuntimeException ae) {
           throw new AssertionError(
-              solrQuery.toString() + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
+              solrQuery + " -> " + rsp.toString() + " ===> " + ae.getMessage(), ae);
```
Comment (cpoerschke) [inline solr/core/src/test/org/apache/solr/search/facet/RangeFacetCloudTest.java:330]:
```suggestion
              solrQuery + " -> " + rsp + " ===> " + ae.getMessage(), ae);
```

**Pull request merged**